doc/gcylcrc.tex

```diff
@@ -77,43 +118,78 @@ \subsubsection{task states to filter out}
 \item {\em default:} runahead
 \end{myitemize}
 
-\subsubsection{dot icon size}
 
-Set the size of the task state dot icons displayed in the text and dot
-views.
+\subsubsection{transpose dot}
+
+Transposes the content in dot view so that it displays from left to right rather
+than from top to bottom. Can be changed later using the options submenu
+avaliable via the view menu.
 
 \begin{myitemize}
-\item {\em type:} string
-\item {\em legal values:} ``small'' (10px), ``medium'' (14px), ``large'' (20px)
-\item {\em default:} ``medium''
+    \item {\em type:} boolean
+    \item {\em legal values:} ``True'' (left-to-right), ``False'' (top-to-bottom)
+    \item {\em default:} ``False''
+    \item {\em example:} \lstinline@transpose dot = True@
 \end{myitemize}
 
-\subsubsection{sort by definition order}
 
-If this is not turned off the default sort order for task names and
-families in the dot and text views will the order they appear in the
-suite definition. Clicking on the task name column in the treeview will
-toggle to alphanumeric sort, and a View menu item does the same for the
-dot view.  If turned off, the default sort order is alphanumeric and
-definition order is not available at all.
+\subsubsection{transpose graph}
+
+Transposes the content in graph view so that it displays from left to right
+rather than from top to bottom. Can be changed later using the options submenu
+avaliable via the view menu.
```

Comment:
spelling - "available" (could just omit the word actually: "using the Options submenu of the View menu"?)